src/utils/richText/useRichText.ts

```diff
@@ -25,6 +25,10 @@ function useRichText(opts: {
   }, [opts.initial]);
 
   const change: RichTextEditorChange = newData => {
+    if (data.current.blocks.length === 0 && newData.blocks.length === 0) {
```

Comment:
Added this because useRichText was triggering setChanged to true after a successful submit with nothing changed in the description